README.md

```diff
@@ -129,46 +129,43 @@ VideoBackground.shared.play(
 
 -> will play the video with the sound on, slightly darkened, and will continuously loop.
 
-> Any combination of the three can be included or left out.
+> Any combination of the parameters can be included or left out.
+
+> For more information about `AVAudioSession`, see the [docs](https://developer.apple.com/library/content/documentation/Audio/Conceptual/AudioSessionProgrammingGuide/AudioSessionBasics/AudioSessionBasics.html#//apple_ref/doc/uid/TP40007875-CH3-SW1).
+
+#### Controls
```

Comment:
Should we add descriptions for these controls/what they do?

Reply:
✅